ampeg.testing/src/test/java/PageObject.java

```diff
@@ -0,0 +1,21 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class PageObject {
+
+	protected WebDriver driver;
+
+	public PageObject(WebDriver driver) {
+		this.driver = driver;
+	}
+
+	public String getURL() {
+		return this.driver.getCurrentUrl();
+	}
+	
+	protected WebElement find(By by) {
```

Comment:
Not sure if this makes things easier. You could replace this approach with PageFactory.InitElements() and it would be cleaner and more descriptive.

Reply:
will work on this